modules/bibdocfile/lib/bibdocfile.py

```diff
@@ -2854,12 +2874,23 @@ def register_download(self, ip_address, version, docformat, userid=0, recid=0):
         docformat = docformat.upper()
         if not version:
             version = self.get_latest_version()
-        return run_sql("INSERT DELAYED INTO rnkDOWNLOADS "
-            "(id_bibrec,id_bibdoc,file_version,file_format,"
-            "id_user,client_host,download_time) VALUES "
-            "(%s,%s,%s,%s,%s,INET_ATON(%s),NOW())",
-            (recid, self.id, version, docformat,
-            userid, ip_address,))
+        if CFG_ELASTICSEARCH_LOGGING:
+            log_entry = {
+                'id_bibrec': recid,
+                'id_bibdoc': self.id,
+                'file_version': version,
+                'file_format': docformat,
+                'id_user': userid,
+                'client_host': ip_address
+            }
+            _DOWNLOAD_LOG.info(log_entry)
+        else:
+            return run_sql("INSERT DELAYED INTO rnkDOWNLOADS "
```

Comment:
Could you please remove the `DELAYED`? see 386ac9d5ef1041f1ac98725e0c3aea178d811f96